### PR TITLE
Added SofastFixed unit tests

### PR DESCRIPTION
**Pull request by braden6521**
Added Sofast Fixed unit tests and unit test data.

### CODE REVIEW

File: opencsp/app/sofast/test/test_SystemSofastFringe.py [hunk 1]
```diff
@@ -14,6 +15,8 @@
 
 @pytest.mark.no_xvfb
 def test_SystemSofastFringe():
+    plt.close('all')
```
Comment (bbean23) [inline opencsp/app/sofast/test/test_SystemSofastFringe.py:18]: Interesting. Why is this needed here?

If this is a fix to plots not being closed in other unit tests, then that is a problem with the other unit tests and should not be spilling over into this integration test.  That being said, for the sake of not blocking progress here, I'd be ok with keeping this with a "TODO".  For example:

```python
plt.close('all') # TODO this is a bandaid, should instead make sure all other tests properly close their plots
```

If these plots are left over from the test_ProcessSofastFixed.py, then maybe add a tear down method.  For example:

```python
@classmethod
def tearDownClass(cls):
    plt.close('all')
```

File: opencsp/app/sofast/test/test_project_fixed_pattern_target.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
 
 @pytest.mark.no_xvfb
 def test_project_fixed_pattern_target():
+    plt.close('all')
```
Comment (bbean23) [inline opencsp/app/sofast/test/test_project_fixed_pattern_target.py:16]: same question here

**Review by bbean23 — DISMISSED** [review]
I would prefer if you respond to my comments first, but otherwise feel free to merge.

File: opencsp/app/sofast/test/test_SystemSofastFringe.py (hunk 1 above)
Comment (braden6521) [reply opencsp/app/sofast/test/test_SystemSofastFringe.py]: As usual, @bbean23, you gave a robust solution that works very well. I implemented the tearDownClass method.

**Review by bbean23 — APPROVED** [review]
Looks good!

**Pull request merged**